components/NewRealmWizard/components/steps/BasicDetailsForm.tsx

```diff
@@ -116,6 +136,20 @@ export default function BasicDetailsForm({
                   data-testid="programId-input"
                   error={errors.programId?.message || ''}
                   autoComplete="on"
+                  success={
+                    !programVersionQuery.isLoading &&
+                    programVersionQuery.data !== 1
+                      ? `Program version ${programVersionQuery.data}`
+                      : undefined
+                  }
+                  warning={
+                    !programVersionQuery.isLoading &&
+                    programVersionQuery.data === 1
+                      ? 'Program version could not be verified'
+                      : programVersionQuery.isFetching
```

Comment:
The difference between `isLoading` and `isFetching` is that `isLoading` is true iff we have no cached data, and so will be true even if no fetching is taking place. `react-query` gives us an answer to every imaginable question we might ask about the state of this query.